istioctl/cmd/istioctl/istioctl_test.go

```diff
@@ -233,6 +245,7 @@ func TestGet(t *testing.T) {
 			`No resources found.
 `,
 			nil,
+			"",
```

Comment:
optional: using named fields might make these test cases easier to parse. You'd also be able to omit unused fields, e.g.

````golang
{
    command: "get routerules",                      
    wantOut: `No resources found. 
 `,	         
}
````